main.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  var clearButton = document.querySelector('#clear-button')
+  var input = document.querySelectorAll('.game-input')
+
+  function enableButton() {
+    clearButton.removeAttribute('disabled')
+    }
+
+  function clearFields() {
+    for (var i = 0; i < input.length; i++) {
+      input[i].value = " "
+    }
+    clearButton.setAttribute('disabled', true)
+  }
+
+  input[0].addEventListener('input', enableButton)
```

Comment:
Kyle is going to change the variable input to something else that isn't already a javascript event title.

Reply:
Fixed